frontend/13f-app/src/components/results.js

```diff
@@ -9,7 +9,7 @@ function Results () {
   const { q, startDate, endDate } = { q: params.get('q'), startDate: params.get('startDate'), endDate: params.get('endDate') }
 
   useEffect(() => {
-    axios.get(`http://localhost:5000/company/search?q=${q}&start_date=${startDate}&end_date=${endDate}`)
+    axios.get(`/company/search?q=${q}&start_date=${startDate}&end_date=${endDate}`)
```

Comment:
could you give it a prefix? maybe like /api/... It will make setting up nginx easier.

Reply:
Environment variable prefix added. Please review.